File: 1816-Truncate-Sentence.js
/**
 * @param {string} s
 * @param {number} k
 * @return {string}
 */
var truncateSentence = function(s, k) {
    let TempArr = s.split(" ");
    TempArr.splice(k);
    
    return TempArr.join(" ");
};

File: 1816-Truncate-Sentence.cs
public class Solution 
{
    public string TruncateSentence(string s, int k) 
    {
        string Result = "";

        string[] TempArr = s.Split(' ');
        string[] SmallArr = new string[k];

        Array.Copy(TempArr, SmallArr, k);

        Result = String.Join(" ", SmallArr);

        return Result;
    }
}
